src/components/snackbar/snackbar.scss

```diff
@@ -10,7 +10,7 @@
 
 .mdc-snackbar__actions {
     .mdc-snackbar__dismiss {
-        color: $mdc-theme-on-secondary;
+        color: rgb(var(--mdc-theme-on-secondary));
```

Comment:
This colour didn't turn out well.

![image](https://user-images.githubusercontent.com/154725/114536204-227f8a80-9c51-11eb-9919-85ee13c72cdc.png)

Reply:
Oh, good catch. I'll check.

Reply:
the reason was that RGB there 👉 `color: rgb(var(--mdc-theme-on-secondary));` 😄 
it should have been `color: var(--mdc-theme-on-secondary);` like the last PR